client/state/reader/tags/items/actions.js

```diff
@@ -9,6 +9,8 @@ import { kebabCase } from 'lodash';
 import {
 	READER_TAGS_REQUEST,
 	READER_TAGS_RECEIVE,
+	READER_UNFOLLOW_TAG_REQUEST,
```

Comment:
i was thinking of tacking on a last PR after landing this series to change all of the action names so that we don't use request/receive anymore.

It'll go by much easier as a standalone and it'll just be a couple renames